tests/30rooms/06invite.pl

```diff
@@ -142,6 +142,14 @@ sub invited_user_can_reject_invite
          die "Expected membership to be 'leave'";
 
       Future->done(1);
+   })->then( sub {
+      matrix_sync( $invitee )
+   })->then( sub {
+      my ( $body ) = @_;
+
```

Comment:
I realise that this would be somewhat unprecedented in this code base, but perhaps a short comment to explain why the dict should be empty might be useful here.